config/app.php

```diff
@@ -149,6 +149,7 @@
         Illuminate\Redis\RedisServiceProvider::class,
         Illuminate\Auth\Passwords\PasswordResetServiceProvider::class,
         Illuminate\Session\SessionServiceProvider::class,
+        // Laravel\Tinker\TinkerServiceProvider::class,
```

Comment:
Tinker is good! un-comment that line.

https://github.com/tightenco/confomo/pull/68/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R17

It already added it as a requirement.